.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ jobs:
                 key: deps1-{{ .Branch }}-{{ checksum "package-lock.json" }}-{{ checksum "package.json" }}-{{ checksum ".circleci/config.yml" }}
             - run:
                 name: Install npm packages
-                command: npm install
+                command: npm ci
```

Comment:
As good a place to start using `npm ci` as any. Provided that the lock file and package.json are aligned, will use the versions from the lock file. Will otherwise fail the build.